### PR TITLE
Add pools to deadline submission

### PR DESCRIPTION
**Pull request by aardschok**
Added support for pools, this allows the artist to set a specific pool of computer to be assigned to the job(s)

* Updated renderglobals instance
* Updated `collect_renderlayer`

Merge this [PR](https://github.com/Colorbleed/colorbleed-config/pull/132) to open support in UI

### CODE REVIEW

File: colorbleed/plugins/maya/publish/collect_renderlayers.py [hunk 1]
```diff
@@ -122,6 +122,15 @@ def parse_options(self, render_globals):
 
         options = {"renderGlobals": {}}
         options["renderGlobals"]["Priority"] = attributes["priority"]
+
+        # Check for specific pools
+        pool_str = attributes.get("pools", None)
+        if pool_str is not None and not "":
```
Comment (BigRoy) [inline colorbleed/plugins/maya/publish/collect_renderlayers.py]: Wouldn't this check exactly the same if you check `if pool_str` :) Doesn't matter really.

Comment (BigRoy) [inline colorbleed/plugins/maya/publish/collect_renderlayers.py]: Wouldn't this check exactly the same if you check `if pool_str` :) Doesn't matter really.

**Comment by BigRoy** [comment]
Nice work! Love the little PRs! :smile:

File: colorbleed/plugins/maya/publish/collect_renderlayers.py (hunk 1 above)
Comment (aardschok) [reply colorbleed/plugins/maya/publish/collect_renderlayers.py]: It makes for less code and less code makes for less things to go wrong ;) lets fix it

**Pull request merged**